sage-android/Sage/app/src/main/java/blueprint/com/sage/models/User.java

```diff
@@ -0,0 +1,31 @@
+package blueprint.com.sage.models;
+
+import java.util.Date;
+
+import lombok.Data;
+
+/**
+ * Created by kelseylam on 10/11/15.
+ */
+public @Data class User {
+    private int id;
+    private Date createdAt;
+    private Date updatedAt;
+    private String email;
+    private String encryptedPassword;
+    private boolean verified;
+    private String firstName;
+    private String lastName;
+    private String authenticationToken;
+    private String role;
+    private int schoolId;
+    private int directorId;
+    private String volunteerType;
+    private int totalHours;
+    private String password;
+    private String confirmPassword;
+
+    // Used for Sign Ups
+    private int schoolPosition = -1;
```

Comment:
keeping track of what option we selected so we can restore it when they go back to the gragment